### PR TITLE
Enhance some stuff in the README

### PR DESCRIPTION
**Pull request by TheFrenchGhosty**
- Add some more badge
- Change some colors
- Add more links / Change some links
- Fix a typo
- Remove the donate section since its now on the website

Rendered version: https://github.com/iv-org/invidious/tree/readme-enhancements#readme

Note: We ideally need to remove more stuff from the README and move it to the website.

### CODE REVIEW

**Comment by SamantazFox** [comment]
Imho, the stars and commits/year counter badges are useless and add clutter.

Also, the `<h1>` shouldn't link to the website since there is a link to the website right below (redundant, and not really acessible).

**Comment by TheFrenchGhosty** [comment]
@SamantazFox I applied most of your suggestions, however I did keep the commits/year badge, because it's not redundant and it's not an information we can get on Github directly, unlike the number of stars.

I also moved the Awesome-Humane-Tech badge to its own line: it looks better since the badge isn't using the same "type of colors" (ie: majority of grey).

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@
   </a>
   <a href="https://github.com/iv-org/invidious/actions">
     <img alt="Build Status" src="https://github.com/iv-org/invidious/workflows/Invidious%20CI/badge.svg">
+  </a>
+    <a href="https://github.com/iv-org/invidious/commits/master">
```
Comment (SamantazFox) [inline README.md:11]:
```suggestion
  <a href="https://github.com/iv-org/invidious/commits/master">
```

Comment (TheFrenchGhosty) [reply README.md]: The editor messed up the indentation oops.

**Review by SamantazFox — APPROVED** [review]

**Pull request merged**